docs/.storybook/configs/theme-context.js

```diff
@@ -4,33 +4,22 @@ import {
   customProperties,
   themesNames,
   ThemeProvider,
-  useTheme,
 } from '../../../design-system';
 
-const defaultThemeName = themesNames.default;
-
 const defaultTheme = customProperties;
 
 const customTheme = {
   colorSolid: '#fff',
   colorSurface: '#1a1a1a',
 };
 
-const ThemeToggler = (props) => {
-  const { applyTheme } = useTheme();
-
-  useEffect(() => {
-    applyTheme({ newTheme: props.name, themeOverrides: props.theme });
-  });
-
-  return null;
-};
-
 const ThemeWrapper = (props) => {
   return (
     <>
-      <ThemeProvider theme={defaultThemeName} />
-      <ThemeToggler {...props} />
+      <ThemeProvider
```

Comment:
Unrelated, but I found we don't need the toggler here so it's better for our future selves maintaining this.